lms/djangoapps/teams/views.py

```diff
@@ -877,6 +877,14 @@ def post(self, request):
         except User.DoesNotExist:
             return Response(status=status.HTTP_404_NOT_FOUND)
 
+        course_module = modulestore().get_course(team.course_id)
```

Comment:
By convention, we usually just call this a `course` instead of a `course_module`.